get.sh

```diff
@@ -41,7 +41,7 @@ DEBUG_IMAGES_REQUIRED=true
 
 usage ()
 {
-	echo 'Usage : get.sh  --testdir|-t optional. path to openjdktestdir. Default value current dir (pwd) is used if not provided'
+	echo 'Usage : get.sh  path to openjdktestdir. Default value current dir (pwd) is used if not provided'
```

Comment:
The message about how to use --testdir should be removed. It should be like:

`	echo 'Usage : get.sh  -platform|-p x64_linux | x64_mac | s390x_linux | ppc64le_linux | aarch64_linux | ppc64_aix'
`

Reply:
updated